src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -161,6 +162,14 @@ relies on script task and is a nice introduction to workflows.
 Scripts can also be used to decorate tasks with specific actions, we support
 <<_pre_post_clean,pre>>, <<_pre_post_clean,post>>, <<_pre_post_clean,clean>> and <<_selection,selection>> scripts.
 
+==== Python Script Engine
+
+Since there are many different versions of Python (mainly Python2 and Python3) which are not compatible, ProActive supports all the versions of python.
+By default, the Python used to execute the Script is the default Python version on your machine. In order to use another Python version to execute
+the task, it is required to put a key 'PYTHON_COMMAND' in Generic Info, and its value is the path (symbolic or absolute, for example 'python3' or
```

Comment:
better to specify Generic Info at task or job level?

Reply:
Done! Good find!